style/style.scss

```diff
@@ -0,0 +1,70 @@
+$color-title:#FF0000;
+$color-subtitle:#B22222;
+$color-name:#A52A2A;
+$background:#6495ED;
+$fontbase:20px ;
+$font-family:"Times New Roman", Times, serif;
+$images:270px;
+$font-title:35px;
+$font-intro:25px;
+$font-subtitle:30px;
+$font-name:28px;
+@mixin flexbox ($justify:center) {
+    display:flex;
+    justify-content:$justify;
+}
+body{
+    background-color:$background;    
+    .titles{
+        color:$color-title;
+        font-size:$font-title;
+    }
+    .subtitle{
+        color:$color-subtitle;
+        font-size:$font-subtitle ;
+    }
+    .intros{
+        font-size:$font-intro;
+        font-family: $font-family;
+    }
+    span{
+        display: block;
+    }
+    main{
+        span{
+            display: block;
+        }
+        .names{
+            color:$color-name;
+            font-size:$font-name;
+        }
+        .descrip{
+            font-size:$fontbase ;
+            font-family:$font-family;        
+            @include flexbox()
+        }
+        .images{
+            width: $images;
+            @include flexbox($justify:flex-end)
```

Comment:
when using the mixin, it's not necessary to declare `$justify` anymore, with `@include flexbox(flex-end)` it's more than enough